package.json

```diff
@@ -11,7 +11,7 @@
     "test:ci": "jest --ci",
     "build": "npm run build:prod",
     "build:dev": "NODE_ENV=development webpack-cli --config webpack/webpack.development.js",
-    "watch:dev": "NODE_ENV=development webpack-cli --config webpack/webpack.development.watch.js & npm run start:dev",
+    "watch:dev": "NODE_ENV=development webpack-cli --config webpack/webpack.development.watch.js & DOCKER_HOST=unix://$HOME/.docker/run/docker.sock npm run start:dev",
```

Comment:
Not sure this can stay in? My docker deamon is not pointed here. Would be interested why this is needed and does the rest of the team need this fix?

Reply:
Agreed, if needed people can export their own env var

Reply:
@naathanbrown I can't run it via docker without having this set in the package json but I just left it in on accident!